Automattic/wp-calypso
# Migrate Domains components CSS to webpack

Migrates styles of components in `components/domains` to webpack.

**How to test:**
Check the rich styles of domain search, both in Signup and in Domains. There is the search bar with filters popover, featured suggestions, normal suggestions, placeholder when search is in progress...

<img width="1034" alt="Screenshot 2019-05-07 at 17 13 03" src="https://user-images.githubusercontent.com/664258/57311181-7390b680-70eb-11e9-8cad-42ab848e05e4.png">

Check the "Already own a domain" flow. Transfer domain to WordPress.com, map domain...

<img width="1061" alt="Screenshot 2019-05-07 at 17 15 23" src="https://user-images.githubusercontent.com/664258/57311410-e7cb5a00-70eb-11e9-91b3-f50836aa9988.png">
